### DISCUSSION TITLE
Hosting Quartz as a Hosted Service inside an ASP.NET Core in Azure Web App

### DISCUSSION THREAD

**Discussion by kotsaris** [quartznet/quartznet#1436] [Q&A] [answered]
Hi

We are looking to include the scheduler in a ASP.NET Core API service which is hosted in Azure Web Apps.

Are there any gotchas (except the always on setting) that we need to be aware of?

**Comment by lahma** [comment, marked as answer]
Basically the only caveat I know is not having the app running via always on or HTTP pings. Have logging enabled at least using warning level to track possible problems.
